auth.go

```diff
@@ -385,22 +383,21 @@ func showAllRoles() {
 }
 
 func showAllIdentities(icentral *imqsauth.ImqsCentral) bool {
-	identities, err := icentral.Central.GetAuthenticatorIdentities()
+	users, err := icentral.Central.GetAuthenticatorIdentities()
 	if err != nil {
 		fmt.Printf("Error: %v\n", err)
 		return false
 	}
-	sort.Strings(identities)
```

Comment:
I know it's a PITA to do Sorts in Go on arbitrary datatypes. As a compromise, at least make our Postgres backend run this query with an "ORDER BY email" clause in it, so that one gets readable output from here.